.github/workflows/verify.yml

```
@@ -44,6 +44,14 @@ jobs:
         run: npm ci
       - name: Run linter
         run: npm run lint
+      - name: Verify file permissions
+        run: |
+          EXE_FILES=$(find icons/ -type f -executable)
+          if test -n "${EXE_FILES-}"
+          then
+            echo "${EXE_FILES}"
```

I think we should add some additional message here to help people understand what's wrong and how to potentially fix this, e.g.:

```suggestion
            echo "Some files were detected to have their executable bit set, see the list below."
            echo "To fix this you can use 'chmod -x icon/FILE_NAME'\n"
            echo "${EXE_FILES}"
```